### PR TITLE
Skip empty files when generating incremental tests

### PR DESCRIPTION
**Pull request by janciesko**
Skip files that do not include "Kokkos_Feature_Level_Required". This includes empty files.

### CODE REVIEW

File: core/unit_test/CMakeLists.txt [hunk 1]
```diff
@@ -337,12 +337,14 @@ FUNCTION (KOKKOS_ADD_INCREMENTAL_TEST BACK_END_NAME)
     IF (NOT CURRENT_TEST_NAME IN_LIST Kokkos_${BACK_END_NAME}_EXCLUDE_TESTS)
        SET (CURRENT_TEST_OUTPUT_FILENAME ${CURRENT_TEST_NAME}_${BACK_END_NAME})
        FILE( STRINGS ${CURRENT_FILE_PATH} CURRENT_REQUIRED_FEATURE_LINE REGEX "Kokkos_Feature_Level_Required" ) 
+       if (NOT ${CURRENT_REQUIRED_FEATURE_LINE} STREQUAL "")
```
Comment (dalg24) [inline core/unit_test/CMakeLists.txt]:
```suggestion
       IF (NOT ${CURRENT_REQUIRED_FEATURE_LINE} STREQUAL "") 
```
As much as I prefer lowercase for CMake, don't break consistency.

Also why did you change this?  It does not feel like a good idea because if one omit or misspell "Kokkos_Feature_Level_Required", the unit test will be silently discarded.

**Comment by crtrott** [comment]
I think we should not do this. As Damien says its dangerous that we suddenly notice we don't actually did the tests. Is there a problem with just adding a level to all those files?

**Comment by crtrott** [comment]
Can we close this?

**Comment by dalg24** [comment]
> Can we close this?

This would be my preference

**Comment by janciesko** [comment]
Agreed. Closing.
